### PR TITLE
Added 'pkg-config' dependency

### PR DESCRIPTION
**Pull request by theduckylittle**
Dep was missing from the developing notes.

### CODE REVIEW

File: DEVELOPING.md [hunk 1]
```diff
@@ -34,7 +34,7 @@ sudo apt-get install -y libcairo2-dev libjpeg8-dev libpango1.0-dev libgif-dev g+
 **MacOS**
 If you have `homebrew` on your machine you can install via
 ```
-brew install cairo
+brew install cairo pkg-config
```
Comment (bartvde) [inline DEVELOPING.md]: what about the other ones?

```brew install pkg-config cairo libpng jpeg giflib```

https://github.com/Automattic/node-canvas/tree/v1.x

Comment (theduckylittle) [reply DEVELOPING.md]: I was seeing libpng, jpeg, and giflib coming in as deps when installing cairo.

Comment (bartvde) [reply DEVELOPING.md]: should we consider just linking to their docs here?

Comment (theduckylittle) [reply DEVELOPING.md]: We did in the next paragraph, I updated the link to match their current documentation structure.

**Comment by bartvde** [comment]
I'm going to do a new release just now, so merging this one in

**Pull request merged**